packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -61,6 +62,14 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
         return localize('Deal cancellation (active)');
     };
 
+    React.useEffect(() => {
+        RudderStack.track('ce_reports_form', {
+            action: 'open_contract_details',
+            form_name: 'default',
+            form_source : 'deriv_trader',
```

Comment:
I suggest not to hard-code these values or at least keep them in a constant file, I can see that `ce_reports_form` is being used 14 times in your PR.